mars/oscar/backends/ray/pool.py

```diff
@@ -36,13 +37,17 @@
 
 @_register_message_handler
 class RayMainActorPool(MainActorPoolBase):
+    @classmethod
+    def process_index_gen(cls, address):
```

Comment:
Invoking thie method multiple times with same address should return same generator

Reply:
Invoking thie method multiple times with same address doesn't return same generator, which is not consistent with `AbstractActorPool.process_index_gen`

Reply:
The implementation of process_index_gen is different between Mars and Ray, it's not a problem.

Reply:
> Invoking thie method multiple times with same address should return same generator

It returns a new generator by address not the same generator.

Reply:
For the caller `AbstractActorPool`, the inferface behaviour should be consistent. We can return same generator for same address or make `AbstractActorPool.process_index_gen` return a new generator for every call